tools/functional-tester/etcd-tester/tester.go

```diff
@@ -96,11 +96,11 @@ func (tt *tester) runLoop() {
 				time.Sleep(time.Second)
 				revs, err := tt.cluster.getRevision()
 				if err != nil {
+					log.Printf("etcd-tester: [round#%d case#%d.%d] failed to get current revisions (%v)", i, j, k, err)
 					if e := tt.cleanup(i, j); e != nil {
 						log.Printf("etcd-tester: [round#%d case#%d.%d] cleanup error: %v", i, j, k, e)
 						return
 					}
-					log.Printf("etcd-tester: [round#%d case#%d.%d] failed to get current revisions (%v)", i, j, k, err)
 					continue
```

Comment:
this continue logic is broken here, which is the root cause of the conflict. We should continue the root for loop (j one), not the sub for loop (k one)